db/schema.rb

```diff
@@ -209,7 +206,6 @@
     t.integer  "position"
     t.datetime "created_at",          null: false
     t.datetime "updated_at",          null: false
-    t.text     "url",                 null: false
```

Comment:
this is breaking tests and galleries. I don't see why it's not in the schema either.

Reply:
Yep, not sure why that got removed. Fixed in c85e8103a827c6827b8d7cc08d6c185a4c3847ef